media/css/firefox/features.less

```diff
@@ -0,0 +1,205 @@
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../sandstone/lib.less";
+
+.fxfeatures-sprite(@posy) {
+    .at2x('/media/img/firefox/features/features-sprite.png', 129px, 214px);
+    background-position: 0 @posy;
+    background-repeat: no-repeat;
+    content: '';
+    display: block;
+    margin: @baseLine auto 0;
+    overflow: hidden;
+    width: 129px;
+}
+
+.mozid {
+    #outer-wrapper {
+        .aurora-bars;
+    }
+
+    #wrapper {
+        overflow: hidden;
+        padding: 0;
+        width: 100%;
+    }
+}
+
+#wrapper {
+    margin-top: 50px;
+}
+
+#main-feature {
+    color: #fff;
+
+    h1 {
+        .font-size(4rem);
+        color: #fff;
+        line-height: 1.2;
+        text-shadow: none;
+
+        @media (max-width: @breakDesktop) {
+            .font-size(2.625rem);
+        }
+
+        @media (max-width: @breakTablet) {
+            .font-size(2.25rem);
+        }
+    }
+
+    p {
+        .font-size(1.75rem);
+        .open-sans-light();
+        line-height: 1.4;
+
+        @media (max-width: @breakDesktop) {
+            .font-size(1.125rem);
+        }
+
+        @media (max-width: @breakTablet) {
+            .font-size(1rem);
+        }
+    }
+
+    h1,
+    p {
+        .span-all();
+        margin-bottom: @baseLine / 2;
+        text-align: center;
+    }
+}
+
+#fxfeature-list {
+    .clearfix();
+    list-style-type: none;
+    margin: 0 auto;
+    padding: 0;
+    width: @breakDesktop - (@baseLine * 2);
+
+    @media (max-width: @breakDesktop) {
+        width: @breakTablet - (@baseLine * 2);
+    }
+
+    @media (max-width: @breakTablet) {
+        width: @breakMobileLandscape - (@baseLine * 2);
+    }
+
+    @media (max-width: @breakMobileLandscape) {
+        width: @breakMobile - @baseLine;
+    }
+
+    li {
+        .transition(background 0.2s ease);
+        background: #fff;
+        background: rgba(255, 255, 255, 0.9);
+        border-radius: 6px;
+        box-shadow: 0 1px 5px rgba(0, 0, 0, 0.2);
+        float: left;
+        margin: 0 (@baseLine) (@baseLine * 2);
+        text-align: center;
+        width: 440px;
+
+        &:hover {
+            background: rgba(255, 255, 255, 0.95);
+        }
+
+        &:nth-child(odd) {
+            clear: left;
+        }
+
+        h2:after {
+            content: '';
+        }
+
+        &.sync h2:after {
+            .fxfeatures-sprite(0);
+            height: 74px;
+        }
+
+        &.private-browsing h2:after {
+            .fxfeatures-sprite(-74px);
+            height: 64px;
+        }
+
+        &.hello h2:after {
+            .fxfeatures-sprite(-138px);
+            height: 76px;
+        }
+
+        @media (max-width: @breakDesktop) {
+            width: 280px;
+        }
+
+        @media (max-width: @breakTablet) {
+            float: none;
+            margin: 0 0 (@baseLine * 2);
+            min-height: 0;
+            width: auto;
+
+            p {
+                min-height: auto;
```

Comment:
`min-height: auto;` can cause trouble in older browsers (it wasn't valid in CSS 2, only recently added in CSS3). `min-height: 0;` is a safer way to override a min-height.